core/assets/js/file_input.js

```diff
@@ -0,0 +1,135 @@
+import { GetText } from "./gettext"
+
+export const FileInputFactory = function(data, locale) {
+  return FileInput(data, locale)
+}
+
+const FileInput = (function(data, locale) {
+    'use strict';
+
+    /* PUBLIC */
+
+    function render(parent){
+      const { title } = data
+      const { description } = data
+
+      const text = {
+        "title": GetText.resolve(title, locale),
+        "description": GetText.resolve(description, locale),
+        "selectButton": GetText.resolve(selectButtonLabel(), locale),
+        "continueButton": GetText.resolve(continueButtonLabel(), locale),
+        "resetButton": GetText.resolve(resetButtonLabel(), locale)
+      }
+
+      parent.el.innerHTML = `
+      <div class="text-title5 font-title5 sm:text-title4 sm:font-title4 lg:text-title3 lg:font-title3 text-grey1">
+        ${text.title}
```

Comment:
Deze moet je denk ik nog escapen. Dat kan evt. via 3rd party lib (https://lodash.com/docs/4.17.15#escape). Een alternatief is om de title en description later als text node te zetten (evt. extra class zodat je ze makkelijk kan selecteren).